packages/flutter/test/material/flat_button_test.dart

```diff
@@ -334,79 +412,209 @@ void main() {
     );
   });
 
-  testWidgets('FlatButton onPressed and onLongPress callbacks are correctly called when non-null', (WidgetTester tester) async {
+  testWidgets('Does FlatButton work with hover', (WidgetTester tester) async {
```

Comment:
It looks like you might have deleted two newer preexisting tests on `flat_button_test.dart` by mistake: 
- 'FlatButton onPressed and onLongPress callbacks are correctly called when non-null'
- 'FlatButton onPressed and onLongPress callbacks are distinctly recognized'

Reply:
Ah, sorry. Accidentally messed up during a merge conflict, but restored it now